libfastpm/thermalvelocity.c

```diff
@@ -237,19 +235,19 @@ fastpm_ncdm_init_create(
     FastPMncdmInitData* nid = malloc(sizeof(nid[0]));
 
     nid->BoxSize = BoxSize;
-    nid->Omega_ncdm = 0.001404; /* FIXME: use new cosmology.c */
+    nid->Omega_ncdm = 0.001404; /* FIX use new cosmology.c */
```

Comment:
vim will automatically highlight FIXME -- FIXME and NOTE are somewhat standardlized comment keywords.